supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -1,3 +1,35 @@
+[discrete]
+[[AArch64]]
+==== image:images/yes.png[yes] AArch64 (noun)
+
+*Description*: 64-bit version of the ARM architecture. Use this term format when referring to OS and server instances. Including: Linux versions, RHEL, or CoreOS. Use uppercase format in general sentences. 
```

Comment:
A 64-bit...

I know, `64-bit` is acting as a noun, but starting with a character reads better than starting a sentence with a number. 

--------------
"Use uppercase format in general sentences." 

Not sure what this applies to? CoreOS would remain as is: a mixture of lowercase or uppercase.